pkg/webhook/core.oam.dev/v1alpha2/application/validating_handler.go

```diff
@@ -68,27 +69,33 @@ func (h *ValidatingHandler) InjectDecoder(d *admission.Decoder) error {
 }
 
 // Handle validate Application Spec here
-func (h *ValidatingHandler) Handle(ctx context.Context, req admission.Request) admission.Response {
+func (h *ValidatingHandler) Handle(_ctx context.Context, req admission.Request) admission.Response {
+	ctx := common.NewReconcileContext(_ctx, types.NamespacedName{Name: req.Name, Namespace: req.Namespace})
```

Comment:
Why validating_handler use new reconcile context, but mutating_handler below still use klog?

Reply:
I will change all of them. It is a bit complex.